client/src/app/components/client-edit/client-edit.component.html

```diff
@@ -1,19 +1,19 @@
-<h1 xmlns="http://www.w3.org/1999/html">Edit client {{ id }}</h1>
+<h1 xmlns="http://www.w3.org/1999/html">{{ 'client-create.create' | translate }} {{ id }}</h1>
```

Comment:
Is correct translation ? For `client-create.create` to translate 'Edit client'?

Reply:
its the same field so i took client-create key, i probably should create a client-edit